bedrock/mozorg/templates/mozorg/about/forums/forums.html

```diff
@@ -73,13 +76,23 @@
         <li id="{{ forum.dashed }}">
           <div class="summary">
             <h4>{{ forum.id }}<a href="#{{ forum.dashed }}">#</a></h4>
-            <p class="description">{{ forum.description|safe }}</p>
+            <p class="description">
+              {%- if forum.id in mdn_forums -%}
+                {{ _('Discussions about MDN Web Docs have moved to Discourse.') }}
```

Comment:
Hmm, the bug does say that the [appropriate forum name](https://bugzilla.mozilla.org/show_bug.cgi?id=1370337#c0) should still be used in each description.

I wonder if another solution may be to remove these old forums from the original source file, and then we could maintain a hard-coded list here in the template. Might that work?